src/fides/api/task/graph_task.py

```diff
@@ -716,6 +743,7 @@ def termination_fn(
             )
         )
         privacy_request.cache_access_graph(format_graph_for_caching(env, end_nodes))
+        privacy_request.cache_data_use_map(_format_data_use_map_for_caching(env))
```

Comment:
When I initially saw this, I felt like this was happening in the wrong place, since it was mimicking the format of something we send to fideslog, that we don't actually use for the access package.

But thinking more, it makes sense to put this here, I think code comments can help clarify.